### PR TITLE
validate_test: add cwd check to TestCheckProcess

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: Zhou Hao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

File: validate/validate_test.go [hunk 1]
```diff
@@ -212,6 +212,38 @@ func TestCheckProcess(t *testing.T) {
...
+			},
+			platform: "windows",
+			expected: specerror.ProcCwdAbs,
+		},
+		{
+			val: rspec.Spec{
+				Version: "1.0.0",
+				Process: &rspec.Process{
+					Args: []string{"sh"},
+					Cwd:  "c:\\foo",
```
Comment (wking) [inline validate/validate_test.go]: Can we get a `windows` test using this `Cwd` with `expected: specerror.NonError` as well?

Comment (q384566678) [reply validate/validate_test.go]: updated.

**Comment by liangchenye** [comment]
LGTM

**Comment by Mashimiao** [comment]
LGTM

**Pull request merged**